HPOBenchExperimentUtils/generate_trajectories.py

```diff
@@ -0,0 +1,136 @@
+"""
+Script to generate `hpobench_trajectory_v1.txt` for completed optimization runs
+"""
+
+import json
+import logging
+import os
+import argparse
+
+from pathlib import Path
+from typing import Union, List, Dict
+
+from HPOBenchExperimentUtils.core.trajectories import create_trajectory
+from HPOBenchExperimentUtils.utils.validation_utils import load_json_files
+
+from HPOBenchExperimentUtils.utils import TRAJECTORY_V1_FILENAME, TRAJECTORY_V2_FILENAME, RUNHISTORY_FILENAME
+
+from HPOBenchExperimentUtils import _log as _root_log
+
+_root_log.setLevel(level=logging.INFO)
+_log = logging.getLogger(__name__)
+
+
+def extract_trajectory(output_dir: Union[Path, str], debug: Union[bool, None] = False,
```

Comment:
isn't this the same as in extract_trajectories.py or why do we need a new function here?